apps/condo/domains/division/components/DivisionForm/CreateDivisionForm.tsx

```diff
@@ -0,0 +1,73 @@
+import { Space, Form } from 'antd'
+import React from 'react'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { useRouter } from 'next/router'
+import { useIntl } from '@core/next/intl'
+import BaseDivisionForm from '../BaseDivisionForm'
+import { Button } from '@condo/domains/common/components/Button'
+import { ErrorsContainer } from '../BaseDivisionForm/ErrorsContainer'
+import { Division } from '@condo/domains/division/utils/clientSchema'
+import { useOrganization } from '@core/next/organization'
+import ActionBar from '@condo/domains/common/components/ActionBar'
+import { get } from 'lodash'
+
+export const CreateDivisionForm: React.FC = () => {
+    const intl = useIntl()
+    const CreateDivisionMessage = intl.formatMessage({ id: 'pages.condo.division.index.CreateDivisionButtonLabel' })
+    const router = useRouter()
+    const { organization } = useOrganization()
+
+    const initialValues = {
+        organization: get(organization, 'id'),
+        responsible: null,
+        properties: [],
+        executors: [],
+    }
+
+    const handleComplete = () => {
+        router.push('/property/')
+    }
+
+    const action = Division.useCreate(initialValues, handleComplete)
+
+    return (
+        <BaseDivisionForm
+            action={action}
+            initialValues={initialValues}
+            organization={organization}
+        >
+            {({ handleSave, isLoading }) => {
+                return (
+                    <Form.Item noStyle dependencies={['properties', 'responsible']}>
+                        {
+                            ({ getFieldsValue }) => {
+                                const { properties, responsible } = getFieldsValue(['properties', 'responsible'])
+
+                                return (
+                                    <ActionBar>
+                                        <Space size={12}>
```

Comment:
antd Space is not treat well with adaptivity out of the box...so...make sure to check this before merging